packages/mobile/src/screens/notifications-screen/NotificationBlock.tsx

```diff
@@ -89,11 +89,11 @@ const typeIconMap: Record<
   [NotificationType.ChallengeReward]: () => IconAudius,
   [NotificationType.TierChange]: (notification: TierChange) =>
     tierInfoMap[notification.tier].icon,
-  [NotificationType.TipReaction]: () => IconTrending,
-  [NotificationType.TipReceived]: () => IconTrending,
-  [NotificationType.TipSent]: () => IconTrending,
-  [NotificationType.TopSupporter]: () => IconTrending,
-  [NotificationType.TopSupporting]: () => IconTrending
+  [NotificationType.Reaction]: () => IconTrending,
+  [NotificationType.TipReceive]: () => IconTrending,
```

Comment:
Why the tense change :eyes:

Reply:
Matches the constants from dn -> identity directly this way